Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/workspace.xml\r\n/vcs.xml\r\n/misc.xml\r\n/modules.xml\r\n/inspectionProfiles/profiles_settings.xml\r\n/inspectionProfiles/Project_Default.xml\r\n/YouTube_Data_Harvesting_and_Warehousing.iml\r\n
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	
+++ b/.idea/.gitignore	
@@ -1,9 +1,2 @@
 # Default ignored files
 /shelf/
-/workspace.xml
-/vcs.xml
-/misc.xml
-/modules.xml
-/inspectionProfiles/profiles_settings.xml
-/inspectionProfiles/Project_Default.xml
-/YouTube_Data_Harvesting_and_Warehousing.iml
